.chloggen/dataset-exporter-export-logs-resource-info-based-configuration.yaml

```diff
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: datasetexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Allow include Logs resource info export to DataSet based on new export_resource_info_on_event configuration. Fix timestamp handling."
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [20660, 23250]
```

Comment:
I noticed that previous changelog entries weren't fully up to date and didn't actually reference related PR number so I went ahead and fixed / updated that as well. 

I also added missing `dataset-exporter-` prefix to one of the changelog files which was missing it (to avoid possible conflicts and similar in the future).